src/app/actions/user.ts

```diff
@@ -0,0 +1,23 @@
+import { Prisma } from '@prisma/client';
+import httpStatusCode from 'http-status-codes';
+
+import * as crypt from '@/lib/crypt';
+import * as userRepository from '@/repositories/user';
+
+export async function addUser(userPayload: Prisma.UserCreateInput) {
+  try {
+    const password = userPayload.password;
+    const cryptedPassword = await crypt.hash(password);
+    const payload = { ...userPayload, password: cryptedPassword };
+    const user = await userRepository.createUser(payload);
+
+    return { data: user, message: 'User is created successfully' };
```

Comment:
How about a more user-friendly message:

```suggestion
    return { data: user, message: 'The user account has been created successfully' };
```

Reply:
Fixed in https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/64/commits/2036f663bcba82365ed0e5fc927d09baace2f2fe